packages/ui/src/number-field/NumberField.css

```diff
@@ -0,0 +1,35 @@
+.ui-NumberField input {
+  border: var(--layout-border-width-medium) solid
+    var(--surface-color-border-dark);
+  border-radius: var(--layout-border-radius-medium);
+  padding: var(--layout-padding-12);
+  font: var(--typography-regular-large-body);
+  width: 100%;
+  max-width: 446px;
```

Comment:
Why 446?

Reply:
That's what the component design was set to 🤷🏻 . Happy to change if Zac wants to take a look and make it different.